integration/devmode/devmode_test.go

```diff
@@ -0,0 +1,277 @@
+		Expect(err).NotTo(HaveOccurred())
+		Expect(docContainers).Should(HaveLen(0))
+
+		By("installing the chaincode")
+		nwo.InstallChaincodeLegacy(network, legacyChaincode, peers...)
+
+		By("instantiating the chaincode")
+		nwo.InstantiateChaincodeLegacy(network, channelName, orderer, legacyChaincode, org1peer0, peers...)
+
+		By("querying and invoking the chaincode")
+		RunQueryInvokeQuery(network, orderer, org1peer0, channelName, 100)
+
+		docContainers, err = client.ListContainers(docker.ListContainersOptions{All: false})
+		Expect(docContainers).Should(HaveLen(0))
+
+		By("killing chaincode process")
+		chaincodeProcess.Signal(syscall.SIGKILL)
+		Eventually(chaincodeProcess.Wait(), network.EventuallyTimeout).Should(Receive())
+
+		docContainers, err = client.ListContainers(docker.ListContainersOptions{All: false})
+		Expect(docContainers).Should(HaveLen(0))
+
+		By("querying and invoking the chaincode")
+		RunQueryInvokeQuery(network, orderer, org1peer0, channelName, 90)
+
+		docContainers, err = client.ListContainers(docker.ListContainersOptions{All: false})
+		Expect(docContainers).Should(HaveLen(1))
+		Expect(docContainers[0].Names[0]).To(ContainSubstring("Org1.peer0-mycc-0.0"))
+	})
+
+	It("executing chaincode in dev mode", func() {
+		chaincode = nwo.Chaincode{
+			Name:            "mycc",
+			Version:         "0.0",
+			Path:            components.Build("github.com/hyperledger/fabric/integration/chaincode/simple/cmd"),
+			Lang:            "binary",
+			PackageFile:     filepath.Join(testDir, "simplecc.tar.gz"),
+			Ctor:            `{"Args":["init","a","100","b","200"]}`,
+			SignaturePolicy: `OR ('Org1MSP.member')`,
+			Sequence:        "1",
+			InitRequired:    true,
+			Label:           "my_prebuilt_chaincode",
+		}
+
+		org1peer0 := network.Peer("Org1", "peer0")
+		peers := network.PeersWithChannel(channelName)
+		orderer := network.Orderer("orderer")
+
+		By("setting up the channel")
+		network.CreateAndJoinChannel(orderer, channelName)
+
+		By("enable new lifecycle capabilities")
+		nwo.EnableCapabilities(network, "testchannel", "Application", "V2_0", orderer, org1peer0)
+
+		By("running the chaincode")
+		chaincodeID := fmt.Sprintf("%s:%s", chaincode.Name, chaincode.Version)
+		peerChaincodeAddress := network.PeerAddress(org1peer0, nwo.ChaincodePort)
+		envs := []string{
+			"CORE_PEER_TLS_ENABLED=false",
+			fmt.Sprintf("CORE_PEER_ADDRESS=%s", peerChaincodeAddress),
+			fmt.Sprintf("CORE_CHAINCODE_ID_NAME=%s", chaincodeID),
+		}
+
+		cmd := exec.Command(chaincode.Path, []string{"-peer.address", peerChaincodeAddress}...)
+		cmd.Env = append(cmd.Env, envs...)
+		chaincodeRunner = ginkgomon.New(ginkgomon.Config{
+			Name:              "chaincode",
+			Command:           cmd,
+			StartCheckTimeout: 15 * time.Second,
+		})
+		chaincodeProcess = ifrit.Invoke(chaincodeRunner)
+		Eventually(chaincodeProcess.Ready(), network.EventuallyTimeout).Should(BeClosed())
+
+		By("approving chaincode for orgs")
```

Comment:
```suggestion
		By("approving the chaincode definition")
```
Nit. Also feels like we should remove "orgs" since we're using a single org network now.

Reply:
fixed